src/main/java/com/code_intelligence/jazzer/junit/FuzzTestExecutor.java

```diff
@@ -201,8 +205,21 @@ static FuzzTestExecutor fromContext(ExtensionContext extensionContext) {
   }
 
   public void addSeed(byte[] bytes) throws IOException {
-    Path seed = Files.createTempFile(javaSeedsDir, "seed", null);
-    Files.write(seed, bytes);
+    Path tmpSeed = Files.createTempFile(javaSeedsDir, "tmp-seed-", null);
```

Comment:
Should we clean up these tmp files? Something like `seed.toFile().deleteOnExit();`?

Reply:
It's moved below, so there is no need to clean it up.

Reply:
`javaSeedsDir` is already temporary, didn't notice that.